setup.cfg

```diff
@@ -1,11 +1,54 @@
+[metadata]
+name = semver
+version = attr: semver.__about__.__version__
+description = attr: semver.__about__.__description__
+long_description = file: README.rst
+author = attr: semver.__about__.__author__
+author_email = attr: semver.__about__.__author_email__
+maintainer = attr: semver.__about__.__maintainer__
+maintainer_email = attr: semver.__about__.__maintainer_email__
+url = https://github.com/python-semver/python-semver
+download_url = https://github.com/python-semver/python-semver/downloads
+project_urls =
+    Documentation = https://python-semver.rtfd.io
+    Releases = https://github.com/python-semver/python-semver/releases
+    Bug Tracker = https://github.com/python-semver/python-semver/issues
+classifiers =
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Software Development :: Libraries :: Python Modules
+license = BSD
+
+[options]
+package_dir =
+    =src
+packages = find:
+python_requires = >=3.6.*
+include_package_data = True
+
+[options.entry_points]
+console_scripts =
+    pysemver = semver.cli:main
+
+[options.packages.find]
+where = src
+
 [tool:pytest]
 norecursedirs = .git build .env/ env/ .pyenv/ .tmp/ .eggs/ venv/
 testpaths = tests docs
 filterwarnings =
     ignore:Function 'semver.*:DeprecationWarning
 addopts =
     --no-cov-on-fail
-    --cov=semver
+    --cov=src
```

Comment:
Unfortunately, your change is not correct. :wink: If I run it with `--cov=src`, the coverage report looks like this:

```
overage.py warning: No data was collected. (no-data-collected)


---------- coverage: platform linux, python 3.6.10-final-0 -----------
Name                        Stmts   Miss    Cover   Missing
-----------------------------------------------------------
src/semver/__about__.py         7      7     0.0%   1-8
src/semver/__init__.py          3      3     0.0%   2-26
src/semver/__main__.py          9      9     0.0%   7-19
src/semver/_deprecated.py      80     80     0.0%   1-378
src/semver/_types.py            7      7     0.0%   1-8
src/semver/cli.py              60     60     0.0%   1-162
src/semver/version.py         230    230     0.0%   1-651
-----------------------------------------------------------
TOTAL                         396    396     0.0%
```

This is obviously wrong. So we definitely need `--cov=semver` here.

Reply:
I changed it back and I increased the coverage while I was at it.